src/main/resources/pl/edu/agh/iet/dts/ui/static/js/controller/userFormController.js

```diff
@@ -2,6 +2,9 @@ app.controller('userFormController', ['$http', '$scope', '$rootScope', '$interva
 
     $scope.submitForm = function () {
         $('#form-panel').css('display', 'none');
+        
+        $interval.cancel($rootScope.promise);
```

Comment:
What if `$rootScope.promise` is not set (i.e. is `null`)? Probably that's not a problem, but make sure about it.